content/docs/setup/kubernetes/spec-requirements/index.md

```diff
@@ -38,3 +38,70 @@ cluster must satisfy the following requirements:
 
 * _**`NET_ADMIN` capability**:_ If pod security policies are enforced in your cluster and unless you use the [Istio CNI Plugin](/docs/setup/kubernetes/istio-cni-install/), your pods must have the `NET_ADMIN` capability allowed.
 See [Required Pod Capabilities](/help/ops/setup/required-pod-capabilities/).
+
+* _**Check required ports**_:
+
+    | Component | Port | NodePort | Description |
```

Comment:
Where did the table come from? How will it be updated? Why are you listing all ports instead of just the ones that can be overridden?

Reply:
Table was generated by me from several places: issues, yaml files, source code.

It would be up to developers to update the table when ports are changed/added.

Istio users would need to make sure all the ports for any services they install are available

I expect as work continues we will allow more ports to be overridable and some to move to a range specific to Istio.

Reply:
thoughts on making this table its own page and link to it from the pod requirement list here?   Seems this is something deserves its own page.

Reply:
The recent incantation of the table was made using `istioctl proxy-config listeners` and merging with data I already had. I also used `istioctl proxy-config enpoint|clusters`.